src/components/Compare/CompareMain.tsx

```diff
@@ -20,7 +25,18 @@ import {
   GeoScopeFilter,
 } from 'common/utils/compare';
 import { Metric } from 'common/metric';
-import * as urls from 'common/urls';
+import { getFirebase } from 'common/firebase';
+
+const firestore = getFirebase().firestore();
+const paramsCollection = firestore.collection('compare-shared-params');
+const nextIdDocRef = paramsCollection.doc('__nextId');
+
+// For filters (0, 50, and 99 are numerical values required by MUI Slider component):
+const scopeValueMap = {
+  [GeoScopeFilter.NEARBY]: 0,
+  [GeoScopeFilter.STATE]: 50,
+  [GeoScopeFilter.COUNTRY]: 99,
+};
```

Comment:
FYI- I moved this stuff to file-level constants because I use them in `useEffect()` and didn't want to have to list them as dependencies that should re-run the effect.